pandas/tests/frame/test_analytics.py

```diff
@@ -301,6 +301,10 @@ def test_count(self):
         expected = Series(0, index=[])
         assert_series_equal(result, expected)
 
+    def test_count_subclassing(self):
+        df = tm.SubclassedDataFrame(data=[[1, 2, np.nan], [2, 3, 4]])
+        tm.assertIsInstance(df.count(), tm.SubclassedSeries)
```

Comment:
I can of course do that, I just thought that correctness of the actual output was tested elsewhere.